qutip/solver/nm_mcsolve.py

```diff
@@ -541,7 +570,15 @@ def _run_one_traj(self, seed, state, tlist, e_ops, **integrator_kwargs):
         result.trace = martingales
         return seed, result
 
-    def run(self, state, tlist, ntraj=1, *, args=None, **kwargs):
+    def run(
+        self,
+        state: Qobj,
+        tlist: Sequence[float],
```

Comment:
In other places, I saw `tlist` hinted as `ArrayLike`. (I don't know which one is better / correct, but it should be consistent, I think.)

Reply:
Neither is exact, `Sequence` description match what we expect, but does not include numpy arrays.
ArrayLike does not limit the type of element or number of dimensions.
I am not sure which we should use...

Reply:
Since we're not aiming to be 100% exact with the hints (as far as I understood), I would think that either is fine as long as it is used consistently.